connector/src/main/scala/com/microsoft/kusto/spark/utils/CslCommandsGenerator.scala

```diff
@@ -55,16 +56,22 @@ private[kusto] object CslCommandsGenerator {
     s".show export containers"
   }
 
-  def generateTableMoveExtentsCommand(sourceTableName: String, destinationTableName: String, batchSize:Int): String = {
-    s""".move extents to table $destinationTableName <|
+  def generateIsTableMaterializedViewSourceCommand(destinationTableName: String): String = {
```

Comment:
Do we have to do two roundtrips to kusto for this? Can't we put this result in a `let` binding, and check it within the command?

Reply:
Not necessarily.

I'm very happy you reviewed this already, but the reason why I didn't yet request your review was because this is my first stab - I haven't even tested it! So this is the brute-force approach, and I intend to optimize once I test and debug.

Again - it's great to have your review even now before it's fully baked, but my intent was that this would be a "draft PR".

Reply:
Unfortunately, it doesn't seem admin commands against the engine can be part of `let` statements.